themes/theme-ios11/components/AppBar/presets/BackBar/connector.js

```diff
@@ -9,4 +21,4 @@ const mapDispatchToProps = dispatch => ({
   goBack: () => dispatch(historyPop()),
 });
 
-export default connect(null, mapDispatchToProps, null, { pure: () => null });
+export default connect(mapStateToProps, mapDispatchToProps, null, { pure: () => null });
```

Comment:
Since this connector now also handles props, the `pure` function needs to be removed. Instead we use the factory as noted above.